src/Driver/Callbacks/Callbacks.jl

```diff
@@ -85,7 +85,10 @@ function GenericCallbacks.call!(cb::SummaryLogCallback, solver, Q, param, t)
         estimated_remaining,
         normQ,
     )
-    isnan(normQ) && error("norm(Q) is NaN")
+    if isnan(normQ)
+        show_not_finite_fields(Q)
+        error("norm(Q) is NaN")
+    end
```

Comment:
```suggestion
    if !isfinite(normQ)
        show_not_finite_fields(Q)
        error("norm(Q) is not finite")
    end
```

Reply:
Not sure if this would be correct.

Reply:
Ah, that was a last minute rush. Definitely want this, good catch!